src/master/master_impl.cc

```diff
@@ -2200,7 +2206,9 @@ void MasterImpl::DeleteTabletNode(const std::string& tabletnode_addr) {
 
 void MasterImpl::TryMovePendingTablets(std::string tabletnode_addr) {
     std::vector<TabletPtr> tablet_list;
-    tablet_manager_->FindTablet(tabletnode_addr, &tablet_list);
+    tablet_manager_->FindTablet(tabletnode_addr,
+                                &tablet_list,
+                                false);  // don't need disabled tables/tablets
```

Comment:
I think the last param should be true.
assume a ts is dead forever; just after it died, a table that the ts's one tablet belongs to is disabled